index.js

```diff
@@ -237,10 +241,11 @@ function addComponent(component) {
   }
 }
 
-var reactAxe = function reactAxe(_React, _ReactDOM, _timeout, conf) {
+var reactAxe = function reactAxe(_React, _ReactDOM, _timeout, conf, _context) {
```

Comment:
Most importantly we're going to need to write a little documentation for this. If you can put something in the readme that would be enough.